### ISSUE TITLE
Manifestation type should be normalized

### ISSUE THREAD

**Issue by J4bbi** [culturesofknowledge/emlo-project#251]
Field `manifestation_type` of `CofkCollectManifestation`

Related issue: https://github.com/culturesofknowledge/emlo-project/issues/198

![image](https://github.com/culturesofknowledge/emlo-project/assets/5488106/dfe9a792-725f-40c7-8f29-fd2ad115d3a6)

The valid mappings are as below

<img width="783" height="389" alt="Image" src="https://github.com/user-attachments/assets/d74b53c7-2698-4a58-998a-fab09612cf65" />

**Comment by MillieG711** [comment]
@anusharanganathan I'm not able to fix this one, it will have to wait for Miranda

**Comment by J4bbi** [comment]
> [@anusharanganathan](https://github.com/anusharanganathan) I'm not able to fix this one, it will have to wait for Miranda

I can fix these. And I'm about to add a check in the uploader validation and only allows a set of values for manifestation types.

**Comment by anusharanganathan** [comment]
@MillieG711 if you are able to map the values shown in the first image to the what you believe to be the correct document_type_code (it's obvious for many of them, but not so for the others), we can create data migrations to correct these values, as per the mapping we have come up with.

**Comment by MillieG711** [comment]
@anusharanganathan I'm afraid I don't know about the non-obvious ones either, I have no idea how that's happened. If you could tell me what the letter IDs are I can tell you what the manifestation type should be, but otherwise I don't know how to find them.

**Comment by jpnp** [comment]
@anusharanganathan I think the way to implement this is as an update (probably via a tweaker script) in the current EMLO-edit DB, rather than in data migration to new EMLO-edit.  Where there are only one or two of a type, then just editing them in the UI seems easiest, as Millie will have to look at the records anyway.

**Comment by anusharanganathan** [comment]
@jpnp good idea. Let’s use the tweaker to modify the obvious ones. For the rest, @J4bbi are you able to produce a list of letters with the manifestations containing the wrong type and attach it to the issue please.

This also will be a nice use case to test the tweaker.
